db/migrate/20240509190359_add_provisioning_columns_to_model_tables.rb

```diff
@@ -0,0 +1,9 @@
+class AddProvisioningColumnsToModelTables < ActiveRecord::Migration[7.1]
+  def change
+    add_column :model_configs, :provisioned, :boolean, default: false, if_not_exists: true
+    add_column :model_servers, :provisioned, :boolean, default: false, if_not_exists: true
+
+    add_column :model_configs, :enabled, :boolean, default: true, if_not_exists: true
+    add_column :model_servers, :enabled, :boolean, default: true, if_not_exists: true
```

Comment:
Oh, this was the flag to avoid deleting the model from the DB.

Can you use `available` instead? When we're autodetecting stuff from Ollama I think we'll use the same column, and `available` is more appropriate for that case while still working for this one, IMO.

Reply:
Oh crap, forgot to change this

Reply:
And done.